src/main/java/com/google/opensesame/servlets/MentorObject.java

```diff
@@ -1,19 +1,43 @@
 package com.google.opensesame.servlets;
 
-public class MentorObject {
-  private String name;
-  private String description;
+import java.util.ArrayList;
 
-  public MentorObject(String nameInput, String descriptionInput) {
-    name = nameInput;
-    description = descriptionInput;
+public class MentorObject extends PersonObject {
+  private ArrayList<String> projectIDs;
+
+  public MentorObject(String nameInput, String gitHubIDInput) {
+    this(nameInput, gitHubIDInput, "", new ArrayList<String>(), new ArrayList<String>());
+  }
+
+  public MentorObject(String nameInput, String gitHubIDInput, String descriptionInput) {
+    this(
+        nameInput,
+        gitHubIDInput,
+        descriptionInput,
+        new ArrayList<String>(),
+        new ArrayList<String>());
+  }
+
+  public MentorObject(
+      String nameInput, String gitHubIDInput, String descriptionInput, ArrayList<String> tagInput) {
+    this(nameInput, gitHubIDInput, descriptionInput, tagInput, new ArrayList<String>());
+  }
+
+  public MentorObject(
+      String nameInput,
+      String gitHubIDInput,
+      String descriptionInput,
+      ArrayList<String> tagInput,
+      ArrayList<String> projectIDInputs) {
+    super(nameInput, gitHubIDInput, descriptionInput, tagInput);
+    projectIDs = projectIDInputs;
   }
 
-  public String getName() {
-    return name;
+  public ArrayList<String> getProjects() {
```

Comment:
```suggestion
  public ArrayList<String> getProjectIds() {
```
nit: this could be more reflective of what is being returned. In the case that you go with my suggestion above, you can update this too.